src/com/akshay/captain/jira.main.kts

```diff
@@ -191,39 +282,159 @@ fun <E> List<E>.ifNotEmpty(function: () -> Unit): List<E> {
     return this
 }
 
-fun createReleaseVersion() {
+/**
+ * Creates a release version in the Jira
+ */
+fun createReleaseVersion(appVersion: String) {
     // create your json here
-
+    val description =
+        "https://biowink.atlassian.net/wiki/spaces/HF/pages/289570878/Android+Releases#AndroidReleases-Android$appVersion.0"
```

Comment:
suggestion: move this string as a constant template on top for easy changing in the future.